src/taipy/templates/multi-page-gui-core/{{cookiecutter.application_name}}/{{cookiecutter.application_main_file}}

```diff
@@ -0,0 +1,66 @@
+from taipy import Core
+from taipy.gui import Gui
+
+from pages.root.root import *
+from pages.scenario_page.scenario_page import scenario_page_md
+from pages.data_page.data_page import data_page_md
+
+from config.config import configure
+
+
+def on_init(state):
+    state.scenario = None
+    state.is_scenario_selected = False
```

Comment:
Useless

Reply:
So we just remove line 12-13 right?

Reply:
Unfortunately, it s more difficult to remove. Please let me handle the simplification since I already did it on locally.